depthai_nodes/ml/messages/base_message.py

```diff
@@ -0,0 +1,15 @@
+from typing import Dict, List, Union
+
+import depthai as dai
+
+
+class MiscelaneousMessage(dai.Buffer):
```

Comment:
Nitpick: I'm not sure about the name. Perhaps `CompositeMessage` (or something similar) fits better since message is composited from other messages?

(in any case I would also rename the file to message name to be consistent with others)